pkg/controllers/user/helm/common/extra_args_test.go

```diff
@@ -0,0 +1,35 @@
+package common
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/pkg/settings"
+	"github.com/rancher/types/apis/project.cattle.io/v3"
+)
+
+func Test_injectDefaultRegistry(t *testing.T) {
+	testRegistry := "test.registry.com"
+	if err := settings.SystemDefaultRegistry.Set(testRegistry); err != nil {
+		t.Fatalf("failed to set system default registry settings, %s", err.Error())
+	}
+
+	testCases := map[string]bool{
```

Comment:
I prefer a struct here so you can name the fields, this will help future engineers understand the test. @daxmc99 has a good example of this style [here](https://github.com/rancher/rancher/blob/master/pkg/node/node_test.go)